### ISSUE TITLE
"Error processing string. Please report"

### ISSUE THREAD

**Issue by Emmi-Eriikka** [HabitRPG/habitica#5025]
_edit by admin:_
### If you experience this problem you do NOT need to comment here or report it anywhere else. We are aware of the problem now and we know it affects everyone.

---

The above came up at my Egg Hunt Quest.. 
I haven't noticed anything bad happening yet but it told me to report so here I am :) 

Here's my API ID: 1352b169-31f0-4503-a9dc-d803a12d752d 

Thanks for all the great work <3

**Comment by harrysalzman** [comment]
I got the same error on my Laundromancer quest.

**Comment by paulashin** [comment]
Yup! Same thing on my "A Stern Talking-To" quest.

**Comment by paglias** [comment]
In which language are you using the site?
Il 14/apr/2015 16:10, "paulashin" notifications@github.com ha scritto:

**Comment by paulashin** [comment]
English

**Comment by stessa-kitsune** [comment]
Hi. 
The issue came up when I joined the "Help! Harpy!" quest on my own. 

This is where it is:
The brave adventurer @UncommonCriminal has disappeared into the forest, following the trail of a winged monster that was sighted several days ago. You are about to begin a search when a wounded parrot lands on your arm, an ugly scar marring its beautiful plumage. Attached to its leg is a scrawled note explaining that while defending the parrots, @UncommonCriminal was captured by a vicious Harpy, and desperately needs your help to escape. Will you follow the bird, defeat the Harpy, and save @UncommonCriminal?
Error processing string. Please report to http://github.com/HabitRPG/habitrpg.
- stessa-kitsune
- Quest Owner
  Rewards

Parrot (Egg)
Parrot (Egg)
Parrot (Egg)
350 Experience
43 Gold
Unlocks purchasable parrot eggs in the Market
To hurt a boss, complete your Dailies and To-Dos. Higher task damage means higher boss damage (completing reds, Mage spells, Warrior attacks, etc). The boss will deal damage to every quest participant for every Daily you've missed (multiplied by the boss's Strength) in addition to your regular damage, so keep your party healthy by completing your Dailies! All damage to and from a boss is tallied on cron (your day roll-over).

Only participants will fight the boss and share in the quest loot.
Abort

Group ID: 97200d73-cc3f-4f2c-ad40-5106320affd7
Name: Myself

**Comment by Emmi-Eriikka** [comment]
Hmm, interesting that so many of us got the same error message.. And in different quests too.
I also use HabitRPG in its basic English form.
I wonder if this is a wider spread problem?

I haven't noticed anything bad happening after the message, though. 
So this shouldn't be such a big problem :) But it is sort of annoying anyway.

**Comment by voidloop** [comment]
Same error message here, in Egg Hunt Quest.
My name: voidloop

**Comment by chrisamiller** [comment]
+1 here. I'm seeing it in an Egg Hunt quest.  UserId: 8bd21466-f3bf-42c4-a4dd-2850a0b062db

![selection_058](https://cloud.githubusercontent.com/assets/48136/7144864/dd037d9e-e2ad-11e4-92ef-c171b8df47f9.png)

**Comment by Alys** [comment]
I think it's coming from this:
common/locales/en/challenge.json:40:    "participants": "<%= membercount %> Participants",
https://github.com/HabitRPG/habitrpg/pull/2966

That string is used in views/options/social/boss.jade as well as for challenges, and membercount has no meaning for quests.

I have to get ready for work and probably won't be able to do a fix for several hours but I can get to it tonight if no one else picks it up before then. Probably a new text string in common/locales/en/quests.json is the easiest way to go. It would be simple enough for a new programmer, so I'll drop this into the Aspiring Coders guild.

**Comment by osmeon** [comment]
I got the same error in the Basi-List quest.  User ID: 455b08ea-d816-4632-a9ed-f42271cda6d2

**Comment by crookedneighbor** [comment]
Alys is correct. I've fixed it and am deploying now. Change should be live in a few minutes.

What caused it was the double use of the participants string in both the challenges and the quests. When we refactored the string to take a variable for number of participants, it broke it on the boss quest page, since that didn't give it a number.

I did a `git grep` through the codebase and confirmed that it was used in no other location.
